src/components/signIn/signin.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { useAppDispatch } from '../../redux/hooks';
+import { email, password } from '../../redux/reducers/signIn';
+const signIn: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const formik = useFormik({
```

Comment:
we discussed - no formic

Reply:
I changed and also delete the sign in to this branch and created another branch.